code/modules/admin/admin_verbs.dm

```diff
@@ -387,6 +392,61 @@ GLOBAL_LIST_INIT(view_runtimes_verbs, list(
 		// TODO: SStgui.on_transfer() to move windows from old and new
 		SSblackbox.record_feedback("tally", "admin_verb", 1, "Aghost") //If you are copy-pasting this, ensure the 2nd parameter is unique to the new proc!
 
+
+/client/proc/admin_observe(mob/target)
+	set category = "Admin"
+	set name = "Aobserve"
+
+	if(!check_rights(R_ADMIN|R_MOD|R_MENTOR))
+		return
+
+	var/actually_admin = check_rights(R_ADMIN|R_MOD)
+
+
+	// todo what happens if a user we're observing logs out?
```

Comment:
Please resolve this TODO.

Reply:
Your hud is only deleted if your mob is destroyed, so this should be safe.